devops/helm/crunchy-postgres/values.yaml

```diff
@@ -0,0 +1,101 @@
+fullnameOverride: simsdb
+appuser: app-database-user
+
+crunchyImage: # it's not necessary to specify an image as the images specified in the Crunchy Postgres Operator will be pulled by default
+#crunchyImage: artifacts.developer.gov.bc.ca/bcgov-docker-local/crunchy-postgres-gis:ubi8-15.2-3.3-0 # use this image for POSTGIS
+postgresVersion: 15
+#postGISVersion: '3.3' # use this version of POSTGIS. both crunchyImage and this property needs to have valid values for POSTGIS to be enabled.
+imagePullPolicy: IfNotPresent
+
+# enable to bootstrap a standby cluster from backup. Then disable to promote this standby to primary
+standby:
+  enabled: false
+  # If you want to recover from PVC, use repo1. If you want to recover from S3, use repo2
+  repoName: repo1
+
+restore:
+  enabled: "${RESTORE_ENABLED}"
+  repoName: repo1
+  options:
+    target: "${RESTORE_TARGET}"
+
+instances:
+  name: ha # high availability
+  replicas: 3
+  dataVolumeClaimSpec:
+    storage: 10Gi
+    storageClassName: netapp-block-standard
```

Comment:
Any reason to change from `netapp-file-standard` to `netapp-block-standard`?
Would the first one not be the recommended one?

Reply:
I did not do any changes from the default values.yml from the bcgov but after your comment, went on researching why this is used. https://docs.redhat.com/en/documentation/openshift_container_platform/4.5/html/scalability_and_performance/optimizing-storage#available-persistent-storage-options_persistent-storage. As per the link given i believe for applications that need full control of storage and operate at a low level on files bypassing the file system. It sounds like our application. If there is any recommendations i would be happy to hear and change it.

Reply:
I have a feeling that the current Patroni may not have the best  config 😉